test/acceptance/tests/basic/terraform/basic-install/main.tf

```diff
@@ -40,6 +40,24 @@ provider "aws" {
   region = var.region
 }
 
+// Generate a gossip encryption key if a secure installation.
+resource "random_id" "gossip_encryption_key" {
+  count       = var.secure ? 1 : 0
+  byte_length = 32
+}
```

Comment:
Sure. If the dev-server generates the encryption key, we'd want to:

* change the dev-server `gossip_key_secret_arn` variable to a flag `enable_gossip_encryption` so we can toggle gossip encryption on/off
* add an output to the dev-server module for the Secret ARN containing the generated encryption key, so it can be passed along to mesh-task(s)

Does that sound right?

Reply:
yep, exactly!

Reply:
I've thought about it some more, and I don't think this will provide any value for us or users since gossip encryption in the dev server is only meant for our acceptance tests, and so it doesn't really matter whether we generate the key in the dev server or acceptance tests.